File: services/diagnostic/src/main/java/org/naima/service/diagnostic/domain/Location.java
package org.naima.service.diagnostic.domain;

public class Location {
    private String locality;
    private String subDistrict;
    private String district;

    public String getLocality() {
      return this.locality;
    }
    public void setLocality(String locality) {
      this.locality = locality;
    }
    public String getSubDistrict() {
      return this.subDistrict;
    }
    public void setSubDistrict(String subDistrict) {
      this.subDistrict = subDistrict;
    }
    public String getDistrict() {
      return this.district;
    }
    public void setDistrict(String district) {
      this.district = district;
    }
}

File: services/cases.net/cases.net/Models/Location.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Org.Naima.Services.Cases.Models
{
    public class Location
    {
        public int Id { get; set; }
        public String Locality { get; set; }
        public String SubDistrict { get; set; }
        public String District { get; set; }

    }
}